Kubernetes/windows/Dockerfile

```diff
@@ -28,5 +28,7 @@ ENV AGENT_VERSION "winakslogprivatepreview"
 ENV OS_TYPE "windows"
 ENV APPLICATIONINSIGHTS_AUTH "OTQzNWI0M2YtOTdkNS00ZGVkLThkOTAtYjA0Nzk1OGU2ZTg3"
```

Comment:
This test AI.  May be intentional. But, I think, we should use Prod AI everywhere.  IIRC, In May agent release, i see there is a task to fix for linux agent also.

Reply:
Yup, this is intentional. This will change to the prod instance once we have the final image we're releasing for public preview